File: problems/problems_836/solution.go
package problem836

import (
	"encoding/json"
	"log"
	"strings"
)

func isRectangleOverlap(rec1 []int, rec2 []int) bool {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var rec1 []int
	var rec2 []int

	if err := json.Unmarshal([]byte(inputValues[0]), &rec1); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &rec2); err != nil {
		log.Fatal(err)
	}

	return isRectangleOverlap(rec1, rec2)
}

File: problems/problems_836/solution.ts
function isRectangleOverlap(rec1: number[], rec2: number[]): boolean {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const rec1: number[] = JSON.parse(inputValues[0]);
	const rec2: number[] = JSON.parse(inputValues[1]);
	return isRectangleOverlap(rec1, rec2);
}
